ios/SafeAreaView/RNCSafeAreaViewEdges.m

```diff
@@ -0,0 +1,29 @@
+#import <React/RCTConvert.h>
+#import "RNCSafeAreaViewEdges.h"
+
+@implementation RCTConvert (RNCSafeAreaView)
+
+RCT_ENUM_CONVERTER(RNCSafeAreaViewEdges, (@{
```

Comment:
Not a fan of this API, what about using something like:

```ts
export type Edges = {
  top?: boolean;
  bottom?: boolean;
  left?: boolean;
  right?: boolean;
};

type Props = {
  disableEdges: Edges;
  // or
  edges: Edges;
};

<SafeAreaView disableEdges={{top: true}} />
// or
<SafeAreaView edges={{top: false}} />
```

Reply:
I don't mind keeping the bitmap as an implementation detail thought.

Reply:
What do you think about `edges={['top', 'bottom']}`?